packages/pyright-internal/src/tests/fourslash/hover.docstring.parameter.fourslash.ts

```diff
@@ -41,6 +41,26 @@
 ////     """
 ////     baz = [|/*marker11*/bar|]
 ////     ...
+////
+//// def foo5([|/*marker12*/bar|]: str, [|/*marker13*/bar2|]: str, [|/*marker14*/bar3|]: str, [|/*marker15*/bar4|]: str) -> None:
+////     """
+////     Foo5 does something
+////
+////     Args:
+////         @param bar: The bar is in town
+////             and is open
+////         :param bar2: The bar is in town
+////             and is open
+////         bar3: The bar is in town
+////             and is open
```

Comment:
maybe these should have different descriptions, just to ensure that each parameter's hover text never matches the wrong description or something
```suggestion
////         @param bar: The bar is in town
////             and is open
////         :param bar2: The bar2 is in town
////             and the bar2 is open
////         bar3: The bar3 is in town
////             and the bar3 is open
```